api/src/main/java/marquez/db/models/ExtendedDatasetVersionRow.java

```diff
@@ -25,12 +25,21 @@ public ExtendedDatasetVersionRow(
       @NonNull Instant createdAt,
       @NonNull UUID datasetUuid,
       @NonNull UUID version,
+      UUID schemaVersionUuid,
```

Comment:
Minor: The `schemaVersionUuid ` can be `null`. I would use `@Nullable ` to communicate that.